### PR TITLE
[1LP][RFR]  Adding new test of edit import dialog. bz_1720617

### PR DESCRIPTION
**Pull request by niyazRedhat**
{{ pytest: cfme/tests/services/test_service_customer_bz.py --use-provider vsphere -vvvv --long-running -k test_edit_import_dialog }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 436151d9726ea2e11d477d458d47de7a99adea2b:

`cfme/automate/dialogs/service_dialogs.py`:
- :bangbang: Line 83:5: **E303** *too many blank lines (2)*

`cfme/tests/services/test_service_customer_bz.py`:
- :bangbang: Line 189:16: **E221** *multiple spaces before operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/services/test_service_customer_bz.py [hunk 1]
```diff
@@ -163,3 +165,29 @@ def test_update_dynamic_checkbox_field(appliance, import_datastore, import_data,
...
+        casecomponent: Services
+        initialEstimate: 1/16h
+        tags: service
+    """
+    sd, ele_label = import_dialog
+    description = fauxfactory.gen_alphanumeric()
+    with update(sd):
+        sd.description = description
+    view = appliance.browser.create_view(DetailsDialogView)
+    view.flash.assert_success_message('{} was saved'.format(sd.label))
```
Comment (ganeshhubale) [inline cfme/tests/services/test_service_customer_bz.py]: *Optional*:
You could use f string here: ```     view.flash.assert_success_message(f'{sd.label} was saved')```

File: cfme/tests/services/test_service_customer_bz.py [hunk 2]
```diff
@@ -163,3 +165,29 @@ def test_update_dynamic_checkbox_field(appliance, import_datastore, import_data,
             "checkbox").checkbox.read() is True and not view.submit_button.disabled, timeout=60)
     except TimedOutError:
         pytest.fail("Checkbox did not checked and Submit button did not enable in time")
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1720617)])
+@pytest.mark.customer_scenario
```
Comment (ganeshhubale) [inline cfme/tests/services/test_service_customer_bz.py]: *Optional*:
You can remove BZ and use it like: ```@pytest.mark.meta(automates=[1720617]```

File: cfme/tests/services/test_service_customer_bz.py [hunk 3]
```diff
@@ -163,3 +165,29 @@ def test_update_dynamic_checkbox_field(appliance, import_datastore, import_data,
             "checkbox").checkbox.read() is True and not view.submit_button.disabled, timeout=60)
     except TimedOutError:
         pytest.fail("Checkbox did not checked and Submit button did not enable in time")
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1720617)])
```
Comment (sbulage) [inline cfme/tests/services/test_service_customer_bz.py]: **Suggestion:** 
@niyazRedhat you can use instead: `@pytest.mark.meta(automates=[1720617])`

File: cfme/tests/services/test_service_customer_bz.py [hunk 4]
```diff
@@ -163,3 +165,29 @@ def test_update_dynamic_checkbox_field(appliance, import_datastore, import_data,
...
+
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        initialEstimate: 1/16h
+        tags: service
+    """
+    sd, ele_label = import_dialog
+    description = fauxfactory.gen_alphanumeric()
+    with update(sd):
```
Comment (sbulage) [inline cfme/tests/services/test_service_customer_bz.py]: **Question:**
@niyazRedhat If you're modifying `description` of the imported dialog, `assert` is not required.?

File: cfme/tests/services/test_service_customer_bz.py (hunk 3 above)
Comment (sbulage) [reply cfme/tests/services/test_service_customer_bz.py]: Ignore this as @ganeshhubale Already reviewed.

File: cfme/tests/services/test_service_customer_bz.py [hunk 5]
```diff
@@ -163,3 +165,30 @@ def test_update_dynamic_checkbox_field(appliance, import_datastore, import_data,
...
+        sd.description = description
+    view = sd.create_view(DetailsDialogView)
+    view.flash.assert_success_message(f'{sd.label} was saved')
+
+    view = navigate_to(sd.parent, "All")
+    assert view.table.row(("Label", sd.label))["description"].text == description
```
Comment (mshriver) [inline cfme/tests/services/test_service_customer_bz.py]: There's a lot going on in this line! :)

**Pull request merged**